pkg/commands/up.go

```diff
@@ -174,11 +184,9 @@ func (o *appfileOptions) apply(ac *v1alpha2.ApplicationConfiguration, comps []*v
 	return application.CreateOrUpdateAppConfig(context.TODO(), o.Kubecli, ac)
 }
 
-func (o *appfileOptions) info(name string) {
+func (o *appfileOptions) info(appName string) {
 	o.IO.Infof("app has been deployed %s%s%s\n", emojiRocket, emojiRocket, emojiRocket)
```

Comment:
try add status check function in https://github.com/oam-dev/kubevela/blob/master/pkg/commands/status.go#L256

It will print route info and other traits info